update textures

Napiophelios opened the pull request:
I have adapted some textures by Gambit for Railcarts for a more conventional look.
HD's are just too bland IMHO.
![pixelbox_railcart by gambit](https://cloud.githubusercontent.com/assets/8119411/14450315/438f3830-004b-11e6-80e6-85c7826ba671.png)

Napiophelios committed:
update textures

Napiophelios committed:
update/upgrade textures
add screenshot
and description text

stujones11 commented:
I would be happy to merge the cart texture but I am not too sure about the changes to the rails or the inclusion of 32px versions.

Napiophelios commented:
Stu the rails were added for my own benefit,
I changed the colors because I have bad eyesight,
didnt really mean/expect for you to merge those :) 

The carts textures are really the only ones that need sprucing up IMHO

I have already added the rail images to my texture pack if anyone want s to use them

stujones11 commented:
> I have already added the rail images to my texture pack if anyone want s to use them

Yes, I think that is best.
